convert-hf-to-gguf.py

```diff
@@ -169,6 +169,30 @@ def from_model_architecture(model_architecture):
         if model_architecture == "PersimmonForCausalLM":
             return PersimmonModel
         return Model
+    
+    @staticmethod
+    def from_model_name(model_name: str):
```

Comment:
I think we should ignore case in this one?

Reply:
To clarify I meant the strings `"StableLM"` should match `"STABLELM"` and `"StableLM"` and `"sTaBlELm"` etc.